backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ProjectDatabaseHandler.java

```diff
@@ -265,6 +265,11 @@ public RequestStatus sendEmailForClearingRequestUpdate(ClearingRequest clearingR
     // GET INDIVIDUAL OBJECTS //
     ////////////////////////////
 
+    public void addSelectLogs(Project project, User user){
```

Comment:
Indentation.

Reply:
Thank you for your review! I will fix them.

Reply:
Fixed it.